python/tvm/relay/prelude.rly

```diff
@@ -0,0 +1,11 @@
+v0.0.1
```

Comment:
Try moving more functions here?

Reply:
We'd need to add ADTs and matching to the text format, which I think would be better left for another PR. It's good to have a simple core merged ahead of the rest, as that will make it easier to iterate afterwards (others can contribute too)